packages/zowe-explorer-ftp-extension/package.json

```diff
@@ -48,9 +48,9 @@
     "vscode": "^1.79.0"
   },
   "dependencies": {
-    "@zowe/zos-files-for-zowe-sdk": "8.0.0-next.202403132009",
-    "@zowe/zos-ftp-for-zowe-cli": "2.1.8",
-    "@zowe/zos-jobs-for-zowe-sdk": "8.0.0-next.202403132009",
+    "@zowe/zos-files-for-zowe-sdk": "8.0.0-next.202403141949",
+    "@zowe/zos-ftp-for-zowe-cli": "3.0.0-next.202403191358",
```

Comment:
Can we remove the "overrides" section from package.json now that the FTP plugin has been updated to v3? 😋

Reply:
fixed in 307fbeb6e